kubeportal/models.py

```diff
@@ -194,12 +197,22 @@ def send_access_request(self, request):
         text_mail = strip_tags(html_mail)
         subject = 'Request for access to "{0}"'.format(settings.BRANDING)
 
-        cluster_admins = User.objects.filter(
-            is_staff=True).values_list('email', flat=True)
+        cluster_admins = []
+
+        if administrator:
+            cluster_admins.append(User.objects.get(username=administrator))
+            logger.info(f"Sending access request from '{self.username}' to '{administrator}'")
+        else:
+            for admin in User.objects.filter(is_superuser=True):
+                cluster_admins.append(admin)
+            logger.info(f"Sending access request from '{self.username}' to all administrators")
+
+        cluster_admin_emails = [admin.email for admin in cluster_admins]
```

Comment:
I was referring to this when I meant line 212. Next time I'll use comments first.